src/GafferArnold/ArnoldDisplacement.cpp

```diff
@@ -200,7 +205,10 @@ bool ArnoldDisplacement::acceptsInput( const Gaffer::Plug *plug, const Gaffer::P
 	{
 		if( const GafferScene::Shader *shader = runTimeCast<const GafferScene::Shader>( inputPlug->source<Plug>()->node() ) )
 		{
-			return runTimeCast<const ArnoldShader>( shader );
+			return
```

Comment:
The "always use {} even for a one-line if" rule isn't really about clarity. It's about leaving the braces out being error prone, especially among a population of coders who are used to using indentation to scope multiline blocks in Python all day long. It's too easy to add a second line and forget the braces in C++.